codeclimate/systems.yaml

```diff
@@ -76,7 +76,9 @@ systems:
 
         meta:
           inputs:
+            - name: org_id
+              description: Code Climate organization ID
```

Comment:
I am thinking this is probably better to set at the sysData level, instead of as a parameter

Reply:
I didn't want to put this on sysData level because it would limit the integration to a single org

Reply:
You can always extend with different ID to create new system. That's the point of a system to keep the system' properties so we don't have to specify it at every call.